spikeinterface_gui/probeview.py

```diff
@@ -86,29 +86,28 @@ def initialize_plot(self):
         #~ self.plot.showAxis('left', False)
         #~ self.plot.showAxis('bottom', False)
     
-        # probe
-        probe = self.controller.get_probe()
-        contact_vertices = probe.get_contact_vertices()
-        planar_contour = probe.probe_planar_contour
-        self.contact_positions = probe.contact_positions
-        
-        # small hack to connect to the first point
-        contact_vertices = [np.concatenate([e, e[:1, :]], axis=0) for e in contact_vertices]
-        
-        vertices = np.concatenate(contact_vertices)
-        connect = np.ones(vertices.shape[0], dtype='bool')
-        pos = 0
-        for e in contact_vertices[:-1]:
-            pos += e .shape[0]
-            connect[pos-1] = False
+        # probes
+        self.contact_positions = self.controller.get_contact_location()
+
```

Comment:
Should there be a warning for overlapping probes? I always just make my probes non-overlapping rather than make my probegroup truly in 3D. But if some has the same x, y points but adds a z shift could there at least be a warning of some sort?